client/signup/steps/p2-site/index.jsx

```diff
@@ -65,10 +66,16 @@ class P2Site extends React.Component {
 			initialState = props.step.form;
 
 			if ( ! isEmpty( props.step.errors ) ) {
+				const errorMessage = props.step.errors[ 0 ].message;
+
+				this.logValidationErrorToLogstash( ERROR_CODE_FROM_LOCAL_STORAGE, errorMessage );
```

Comment:
should we log the entire `props.step.errors`? maybe it contains some useful information on future debugging

Reply:
We could but `logValidationErrorToLogstash` probably won't be able to handle an array right now. I'm not sure it could contain more than errors for `site` and `siteTitle` and since we only hydrate `site` here I'd say it's okay to log just that.